cinema/serializers.py

```diff
@@ -1,6 +1,39 @@
 from rest_framework import serializers
+from cinema.models import Movie, Genre, Actor, CinemaHall
 
-from cinema.models import Movie
+
+class GenreSerializer(serializers.Serializer):
```

Comment:
Consider using `serializers.ModelSerializer` instead of `serializers.Serializer` for `GenreSerializer`. This will automatically generate the `create` and `update` methods and reduce boilerplate code.